autoendpoint/src/routers/apns/router.rs

```diff
@@ -268,21 +317,136 @@ impl Router for ApnsRouter {
             .clients
             .get(channel)
             .ok_or(ApnsError::InvalidReleaseChannel)?;
-        let payload = Payload {
-            aps,
-            data: message_data
-                .into_iter()
-                .map(|(k, v)| (k, Value::String(v)))
-                .collect(),
-            device_token: token,
-            options: NotificationOptions {
+        let mut aps = Self::default_aps();
```

Comment:
Sorry, I probably missed some this the last time I reviewed it (is James back from PTO yet, can we just kill all this? 😄 ) -- looking at it now, I think we can at least move this large chunk into its own method, I've taken a [shot at that on this branch](https://github.com/mozilla-services/autopush-rs/compare/chore/main-a2...chore/main-a2-move-to-method).

Reply:
I mean, sure? It's still a big ugly pile of code regardless of if we want to screw with the stack or not. Tossing it into a different function doesn't really help, and IDEs allow for block collapse if we just want to ignore it for now.

Reply:
It's just too many lines for one method, this block is > 100 by itself (clippy's pedantic `too_many_lines` lint is that). I assumed it was in here due to lifetime trickyness, hence my branch to solve that w/ the new method. I wouldn't justify e.g. an even larger say 500 line method either with IDE collapsing help -- so I'll argue this does help readability.

Reply:
Yeah, fair. I broke it out into it's own function like you suggested. (changed the name and gave it a better comment) I was debating how much of that I could reduce using macros, but I figured that was more work for what was (hopefully) a temporary thing.